test/ILLink.RoslynAnalyzer.Tests/RequiresAssemblyFilesAnalyzerTests.cs

```diff
@@ -3,22 +3,22 @@
 using Microsoft.CodeAnalysis.Testing;
 using Xunit;
 using VerifyCS = ILLink.RoslynAnalyzer.Tests.CSharpAnalyzerVerifier<
-	ILLink.RoslynAnalyzer.SingleFileUnsupportedAnalyzer>;
+	ILLink.RoslynAnalyzer.RequiresAssemblyFilesAnalyzer>;
 
 namespace ILLink.RoslynAnalyzer.Tests
 {
-	public class SingleFileUnsupportedAnalyzerTests
+	public class RequiresAssemblyFilesAnalyzerTests
 	{
-		static Task VerifySingleFileUnsupportedAnalyzer (string source, params DiagnosticResult[] expected)
+		static Task VerifyRequiresAssemblyFilesAnalyzer (string source, params DiagnosticResult[] expected)
 		{
 			// TODO: Remove this once we have the new attribute in the runtime.
 			source = @"namespace System.Diagnostics.CodeAnalysis
 {
 #nullable enable
 	[AttributeUsage (AttributeTargets.Method | AttributeTargets.Constructor, Inherited = false, AllowMultiple = false)]
-	public sealed class SingleFileUnsupportedAttribute : Attribute
+	public sealed class RequiresAssemblyFilesAttribute : Attribute
```

Comment:
Are we changing the attribute to match the [new proposal](https://github.com/dotnet/runtime/issues/45705#issuecomment-755776365) with the parameterless constructor and extended attribute usage?